ci/docs.yaml

```diff
@@ -0,0 +1,20 @@
+name: docs
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - astropy>=4.0
+  - astropy-healpix
+  - h5py
+  - numpy
+  - h5py
+  - scipy
+  - coverage
+  - pypandoc
+  - pytest-cov
+  - setuptools_scm
+  - sphinx
+  - pyuvdata>=2.0.1
+  - pip
+  - pip:
+      - git+https://github.com/aelanman/lunarsky.git
```

Comment:
`lunarsky` is now on pypi, if you want to change the installation path.

Reply:
ok, I'll fix that.

Reply:
I actually went further and added it in `extras_require` and the readme to highlight it a bit better.